### PR TITLE
WIP: Add reconciler to knativeserving

### PR DESCRIPTION
**Pull request by houshengbo**
Closes: #60

### CODE REVIEW

File: pkg/reconciler/knativeserving/knativeserving.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	// Listers index properties about resources
+	knativeServingLister          listers.KnativeServingLister
+}
+
+// Check that our Reconciler implements controller.Reconciler
+var _ controller.Reconciler = (*Reconciler)(nil)
+
+func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
+	return nil
+}
```
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/knativeserving/knativeserving.go]:
```suggestion
}

```

Comment (mattmoor-sockpuppet) [inline pkg/reconciler/knativeserving/knativeserving.go]:
```suggestion
}

```

**Comment by k4leung4** [comment]
looks like the cla bot is happy now

**Comment by k4leung4** [comment]
nm, wrong pr

File: pkg/reconciler/reconciler_test.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 The Knative Authors
```
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/reconciler_test.go]:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/reconciler/stats_reporter_test.go [hunk 3]
```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2018 The Knative Authors
```
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/stats_reporter_test.go]:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/reconciler/knativeserving/controller.go [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+		}
+		if err == nil {
+			err = manifest.Transform(mf.InjectNamespace(operand))
+		}
+		if err == nil {
+			err = manifest.ApplyAll()
+		}
+	}
+	return
+}
```
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/knativeserving/controller.go]:
```suggestion
}

```

File: pkg/reconciler/reconciler_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/reconciler_test.go]:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/reconciler/stats_reporter_test.go (hunk 3 above)
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/stats_reporter_test.go]:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/reconciler/reconciler_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/reconciler_test.go]:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/reconciler/stats_reporter_test.go (hunk 3 above)
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/stats_reporter_test.go]:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/reconciler/knativeserving/controller.go [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
```
Comment (mattmoor-sockpuppet) [inline pkg/reconciler/knativeserving/controller.go]:
```suggestion
}

```

Comment (mattmoor-sockpuppet) [inline pkg/reconciler/knativeserving/controller.go]:
```suggestion
}

```

**Pull request closed without merging**